src/main/java/bobthebot/utils/Ui.java

```diff
@@ -0,0 +1,132 @@
+package bobthebot.utils;
+
+import bobthebot.tasks.Task;
+import bobthebot.tasks.ToDoList;
+
+/**
+ * Class which handles the interaction of BobTheBot and the user.
+ * */
+public class Ui {
+    /**
+     * Method which welcomes the user.
+     * */
+    public static void welcome() {
+        String result = "\tHello! I am Bob the Bot, your friendly task manager! \uD83D\uDE0A\n";
+        result += "\tWhen using me, please stick to the following commands:\n"
+                + "\t\t1. todo - for items that you have to do\n"
+                + "\t\t2. deadline - for items which have an upcoming deadline\n"
+                + "\t\t3. event - for events with a date and time\n"
+                + "\n"
+                + "\t\t4. mark - to mark an event as done\n"
+                + "\t\t5. unmark - to mark an event as undone\n"
+                + "\t\t6. delete - to delete an event\n"
+                + "\t\t7. list - to view all the events on your todo list\n"
+                + "\t\t8. bye - to wish me a (temporary) farewell";
+
+        formatMessage(result);
+    }
+
+    /**
+     * Method which says goodbye to the user.
+     * @param list ToDoList of the items the user still has to accomplish.
+     * */
+    public static void goodbye(ToDoList list) {

```

Comment:
```suggestion
    public static void sayGoodbye(ToDoList list) {
```
Perhaps this can be slightly changed from a noun to be a verb since it is a method?